openmeetings-web/src/main/webapp/css/raw-wb.css

```diff
@@ -45,7 +45,8 @@ html[dir="rtl"] .room-block .sb-wb .wb-block {
 	position: relative;
 	padding-right: 25px;
 }
-.room-block .sb-wb .wb-block .tabs .wb-tabbar li a button {
+.wb-tab-close {
```

Comment:
I think we should really start shorting those CSS paths. Having those long selectors is really difficult to manage. For example if any of those elements in the path ".room-block .sb-wb .wb-block .tabs .wb-tabbar li a button"
would either change, rename or any order changes, this CSS selector would stop working.
It is not a good practise to have long CSS selectors.
See for example: https://stackoverflow.com/questions/14519115/is-having-a-long-css-selector-bad

Its not a performance problem but just makes the code very hard to change.

Reply:
This is true
On the other hand short selectors are often not specific enough :(

As I wrote before: let's release then can start refactor (too much testing otherwise)

Reply:
well that is why you add classes to them, so you can reference them uniquely. Or by their "type" of thing.

Reply:
Unfortunately it will not work in some cases

Here is an example:

- imagine you would like to style bootstrap tab
- you add `unique` class to it (let it be `om-2251-test`) and define it as follows:
```
.om-2251-test {
    color: red;
}
```
- and ....  the color is `black`

The reason is the color is set in rule `.nav-tabs .nav-link.active` which is more specific

This is why every optimization like this need to be carefully tested :((

Reply:
That seems more like a bootstrap issue.

The recommendation in this case seems to be you overwrite the bootstrap CSS in a custom css file and give it the exact same name like in original the bootstrap css definition and make sure the custom style gets loaded after the default:
https://uxplanet.org/how-to-customize-bootstrap-b8078a011203

So if struggling with bootstrap styles you just overwrite those. 

And obviously not try to use the same bootstrap element in different ways in different places. But rather come up with new base style.

Reply:
Well, this was very simple example :)
Usually you are creating CSS class and expect it to work with multiple elements (do-not-repeat-yourself), but often it is not possible
So I end up with long bullet-proof selectors :)))

I am +1 to simplify those in M5 (or maybe it will be `5.0.0` without-M)
Additionally I have in plans to use Chrome CSS analyser to remove unused styles :))